### ISSUE TITLE
Installation without neural network dependencies

### ISSUE THREAD

**Issue by braaannigan** [autogluon/autogluon#2868]
Hi - great work on the library, I'm impressed how far it has come on in the last few years (particularly on inference speed).

I can see for my timeseries forecasting problem that AutoGluonTabular works really nicely in terms of performance, training speed and inference speed.

I'd like to avoid having to install torch and nvidia-cuda. Can I run without these given i am setting AutoGluonTabular as the only model to train in my hyperparameters? If so, can I install autogluon without having to install these large dependencies?

**Comment by Innixma** [comment]
Yes, refer to https://auto.gluon.ai/stable/index.html#installation

For example `pip install autogluon.tabular[xgboost,lightgbm,catboost]` will not install torch, and not train the neural network models.

> "given i am setting AutoGluonTabular as the only model to train in my hyperparameters?"

I don't understand what you mean. AutoGluon Tabular is an ensemble of models that includes torch models. If you don't care about the torch models in the ensemble, then you don't need torch. If you care about them as they improve performance of the ensemble, then you want torch.

If you want to avoid installing CUDA, then follow the above install guide and select `CPU` to get CPU install instructions that avoid CUDA.

**Comment by braaannigan** [comment]
@Innixma Thanks for your reply, sorry I missed the notification. 

I hadn't realised AutoGluon Tabular includes torch models (I assumed it was just the gradient boosting models), I'll have a look at the model zoo in more detail.

I'll try your suggestions and close the issue if it looks good
